src/auth/views/authorize.pug

```diff
@@ -0,0 +1,14 @@
+html
```

Comment:
C'est quoi le pug file et surtout pourquoi l'avoir utilisé ici ?

Reply:
The default template engine of NodeJS (before it was Jade). Here we need to return a ugly HTML page to the (external) clients which want to use our API. Inside those clients, the user needs to click on the "approve" button to give access to the client to our API.